editor/src/clj/editor/editor_extensions.clj

```diff
@@ -601,7 +665,8 @@
                                 (future/then (reload-resources!) rt/and-refresh-context))
                :out (line-writer #(display-output! :out %))
                :err (line-writer #(display-output! :err %))
-               :env {"editor" {"get" (make-ext-get-fn project)
+               :env {"editor" {"bundle" {"project_binary_name" ext-project-binary-name} ;; undocumented, hidden API!
```

Comment:
`editor.bundle` module is internal. It exists in the runtime, but the docs/completions are displayed only when running the editor in dev. This will allow us to change the bundling in the future if we decide to change the bundling UI/UX.